src/cmd/create.go

```diff
@@ -390,7 +396,7 @@ func createContainer(container, image, release string, showCommandToEnter bool)
 	createArgs = append(createArgs, xdgRuntimeDirEnv...)
 
 	createArgs = append(createArgs, []string{
-		"--hostname", "toolbox",
+		"--hostname", createFlags.hostname,
```

Comment:
We can't directly use `createFlags.hostname` in `createContainer` because `src/cmd/run.go` also uses this function.